src/fetchProjectItems.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  GetProjectsDocument,
+  GetProjectsQuery,
+  GetProjectsQueryVariables, Label,
+} from "./generated/graphql-operations";
+import {ProjectDetails, ProjectItem, ProjectItemLabels} from "./types";
+import { print } from "graphql/index";
+import { octokit } from "./octokit";
+
+export async function fetchProjectItems(
+  params: GetProjectsQueryVariables,
+  recursiveCollection?: ProjectDetails
+): Promise<ProjectDetails> {
+  console.log(`Fetching projects`, params)
+  // @ts-ignore
+  const project: GetProjectsQuery = await fetchProject(params).catch((e) => {
```

Comment:
On line 100 the `fetchProject` implementation already logs&exits, so is this catch actually needed?